models/ed/R/write.configs.ed.R

```diff
@@ -447,7 +447,7 @@ write.config.jobsh.ED2 <- function(settings, run.id){
   } else {
     modeloutdir <- file.path(settings$run$host$scratchdir, settings$workflow$id, run.id)
     mkdirscratch <- paste("mkdir -p", modeloutdir)
-    copyscratch <- paste("rsync", "-a", paste0('"', file.path(modeloutdir, "*"), '"'), paste0('"', outdir, '"'))
+    copyscratch <- paste("rsync", "-a", paste0('"', file.path(modeloutdir, ""), '"'), paste0('"', file.path(outdir, ""), '"'))
```

Comment:
OK, actually I guess no since putting the star there made the run fail again....

```
Loading required package: stringr
> model2netcdf.ED2('/data/sserbin/Modeling/ED2_Modeling/US-WCr/PEcAn.US-WCr/ED2.git.run/pecan.26/out/ENS-00001', 45.92, -90.45, '2003/06/15', '2005/01/01')
Loading required package: udunits2
[1] "*** WARNING: No tower output for : /data/sserbin/Modeling/ED2_Modeling/US-WCr/PEcAn.US-WCr/ED2.git.run/pecan.26/out/ENS-00001"
Error in model2netcdf.ED2("/data/sserbin/Modeling/ED2_Modeling/US-WCr/PEcAn.US-WCr/ED2.git.run/pecan.26/out/ENS-00001",  :
  no loop for break/next, jumping to top level
Execution halted

```

Switching back but I need to sort out why it now fails with read output even though the netCDF files are there.